src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -254,50 +255,78 @@ class C {
         [Fact]
         public async Task OpenStringLiteral()
         {
-            await VerifyItemIsAbsentAsync(AddUsingDirectives("using System;", AddInsideMethod("string s = \"$$")), @"String");
-            await VerifyItemIsAbsentAsync(AddUsingDirectives("using System;", AddInsideMethod("string s = \"$$")), @"System");
+            var code = AddUsingDirectives("using System;", AddInsideMethod("string s = \"$$"));
+            var result = ImmutableArray.Create(
+                CompletionTestExpectedResult.Absent("String"),
+                CompletionTestExpectedResult.Absent("System")
+            );
+            await VerifyExpectedItemsAsync(code, result);
```

Comment:
```suggestion
            await VerifyExpectedItemsAsync(code, [
                CompletionTestExpectedResult.Absent("String"),
                CompletionTestExpectedResult.Absent("System")
            ]);
```

And so on and so forth.

Reply:
I must definitely get used to collection expressions. You may cancel the current check runs.